coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/loader/sql/AbstractRepBasedDataSqlizer.scala

```diff
@@ -11,8 +11,17 @@ import com.socrata.datacoordinator.id.{ColumnId, RowId}
 import com.socrata.datacoordinator.util.collection.{ColumnIdSet, RowIdSet, MutableRowIdSet}
 import java.io.Closeable
 
+/**
+ * Abstract SQL adapter for data access
+ *
+ * @param softMaxBatchSize in bytes, the max size of each batch to insert.
+ * This is actually very tricky because the size is estimated from the SqlRep's estimatedSize
+ * method for each column value, but doesn't include the intermediate objects produced, which
+ * are actually the most expensive and may take 100x as much memory as the estimated size.
```

Comment:
It's only vaguely "in bytes"; the intent isn't to give an exact (or even really very rough) value so much as a vague "this is big, this is small, we probably have a lot of data and should flush it to the DB now" estimate.

Maybe it should be `@param softMaxBatchSize in story points`